src/coreComponents/dataRepository/CMakeLists.txt

```diff
@@ -59,8 +59,6 @@ blt_add_library( NAME       dataRepository
 
 target_include_directories( dataRepository PUBLIC ${CMAKE_SOURCE_DIR}/coreComponents )
 
-geosx_add_code_checks( PREFIX dataRepository )
```

Comment:
Why did you remove all of these calls?

Reply:
I think they are unnecessary to create all these component level targets.